### DISCUSSION TITLE
Error: Sandbox exited with code 0, but produced no response baton

### DISCUSSION THREAD

**Discussion by huyjs9** [vercel/community#1882] [Help] [answered]
### Summary

I am using GitHub Actions with Vercel CLI for the production deployment.
Here is the reference document: https://vercel.com/guides/how-can-i-use-github-actions-with-vercel
When GitHub Actions runs to the step `Deploy Project Artifacts to Vercel`, I received an `Error: Sandbox exited with code 0, but produced no response baton`
The command I used in this step:
```bash
vercel deploy --prod --prebuilt --token=***
```

This is the result:
```bash
Vercel CLI 28.18.1
Retrieving project…
Deploying workspace/project-name
Uploading [--------------------] (0.0B/9MB)
Uploading [=====---------------] (2.3MB/9MB)
Uploading [==========----------] (4.5MB/9MB)
Uploading [===============-----] (6.8MB/9MB)
Uploading [====================] (9.0MB/9MB)
Inspect: https://vercel.com/*** [11s]
Queued
Building
Error: Sandbox exited with code 0, but produced no response baton
https://***.vercel.app/
Error: Process completed with exit code 1.
```

### Additional information

After running the deployment again, it worked properly.

**Comment by amyegan** [comment, marked as answer]
I'm glad the deployment was successful on the second try. Sorry you ran into an error the first time. This isn't enough information to be able to determine the cause of the error, but you may be able to find more details in your project's [Build Logs](https://vercel.com/docs/concepts/deployments/logs#)
